shell.nix

```diff
@@ -16,6 +16,7 @@ in with pkgs;
       hlint
       haskellPackages.apply-refact
       stylish-haskell
+      git
```

Comment:
nice, I use the zsh integration so I am always in an impure shell. I tend to forget these important deps

Reply:
I tried to use a pure shell but there was a problem with libz. But yeah, git was missing :-P